en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -72,17 +75,22 @@ If using a Spektrum satellite receiver, this should be connected to the flight c
 
 ## Power
 
-The V5+ kit includes the HV\_PM, which supports 2~10s LiPo battery.
-Please connect the 6pin connector of the HW\_PM module to the flight control `Power1` interface.
+The V5+ kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+Connect the 6pin connector of the *HW\_PM* module to the flight control `Power1` interface.
+
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_plus.md#issue_pm_unfused)).
```

Comment:
FYI, again link is to the issue about unfused power module.

Reply:
@hamishwillee V5+ is different. It has over current protection on the 5 Volt Peripheral and 5 Volt high power. This limits the current to 2.5A but has short circuit protection.
 Therefore 2.5 A can be delivered to the connectors with V5+ listed as pin 1 (that are rated at 1 A). This still can pose a potential danger. Please make it Clear 1) that power needs to be off while connecting peripherals . 2) Improper wiring can lead to equipment damage or personal harm.

Reply:
Thanks @davids5.

I added a very similar warning everywhere, which I hopes works for you. For the wiring quickstarts this looks like:
![image](https://user-images.githubusercontent.com/5368500/61923124-29aac480-afa6-11e9-9bbc-74a1acf1fbe6.png)
For the FC docs "issue section" they both look like:
![image](https://user-images.githubusercontent.com/5368500/61923155-4e9f3780-afa6-11e9-8bf1-06dce69102af.png)

I did not add information about the differences in current protection on the two boards you refer to above - because from an issue point of view the direction is the same.

But the information might be useful to someone? Would it make sense to have a section "Current Protection" below *Voltage Ratings* that for:
- nano says "There is no current protection"
- v5+ say "The board has over current protection on the 5 Volt Peripheral and 5 Volt high power. This limits the current to 2.5A but has short circuit protection.
Up to 2.5 A can be delivered to the connectors with V5+ listed as pin 1 (that are rated at 1 A).

Reply:
PS If this is not "quite right" but is an improvement then let me know whether you are OK with submitting as is. Happy to iterate forever, but would prefer to constrain changes and get warnings out quickly.

Reply:
Change:nano says "The Nano has no over current protection"

Change:"The V5+ has over current protection on the 5 Volt Peripheral and 5 Volt high power. This limits the current to 2.5A and has short circuit protection.

Caution:Up to 2.5 A can be delivered to the connectors with V5+ listed as pin 1 that are only rated at 1 A.

Reply:
Thanks, for the V5+ case the English for this sentence wasn't quite right: "This limits the ....". You can see how I modified this here: https://github.com/PX4/px4_user_guide/pull/538/commits/e775587a70a75c77e53a9ee13eb333ae64c05d11